### DISCUSSION TITLE
Thank you, service, thank you.

### DISCUSSION THREAD

**Discussion by infott2t** [react-navigation/react-navigation#9322] [General]
Let's safely use smartphones, etc.

Say, React Native. About. Thank you.

Tag, like making, function, so need a return to have, props. and const word, using.

so, 

export default App;

last word, coding. and App. It's function or Tag. The user making HTML TAGS.

so,

> const App () => {return();}

and, some making Tag too.

>const ATag() => {return();}   

and or, props use, 

> const ATag(props) => {return();}

props, like this.  For example,

>`<div id="styleBottom" name="catty"></div>`

properties, on HTML.

so, 

ATag use in App,

> const ATag (props) =>{
>
> return ( `<Text> {props.stringName} </Text>`);
>}
>
> const App() => {
> return ( 
> `<ScrollView>`
> ` <Text>App~.</Text>`
> `<ATag stringName="ATagProps"></ATag>`
> `<View>`
> `<Text>Hello</Text>`
> `</View>`
> `</ScrollView>` );
> }
>
> export default App;

Something, Quote, bad but understanding, good?

Reference, [https://reactnative.dev/docs/intro-react](https://reactnative.dev/docs/intro-react)  
Let's study. Hope good development. 

How about this~. Make Tag, and all. props. and after learning state, after, using react-navigation.

Let's safely use smartphones, etc.

 
How about church~. Jesus Christ. Phone calling to church~. Let's go there. I like this say. 

Thank you. Watching. Thank you.

**Comment by Yacubane** [comment]
don't do drugs

**Reply by infott2t** [reply]
Thank you, comment, thank you. 

Have a good day~. 

Let's safely use smartphones, etc.

How about church~. Jesus Christ. Phone calling to church~. Let's go there. I like this say.

Thank you. Watching. Thank you.
